lib/rex/post/hwbridge/ui/console.rb

```diff
@@ -75,7 +75,7 @@ def interact(&block)
   #
   # Interacts with the supplied channel.
   #
-  def interact_with_channel(channel)
+  def interact_with_channel(channel, *_)
```

Comment:
```suggestion
  def interact_with_channel(channel, **_)
```

https://www.ruby-lang.org/en/news/2019/12/12/separation-of-positional-and-keyword-arguments-in-ruby-3-0/

Reply:
I believe it will still work without this change, but might as well be explicit with our intent :+1: